Assets/Scripts/Dungeon/ProcGen/MapGen/DBM.cs

```diff
@@ -44,7 +45,7 @@ public void AddCell()
         /// <summary> Converts the DBM cell pattern into a list of rooms. </summary>
         /// <param name="roomTemplet"> The templet Room GameObject used for instantiation. </param>
         /// <returns> The DBM pattern as an array of rooms. </returns>
-        public Room[] GetPatternAsRooms(Room roomTemplet)
+        public Room[] GetPatternAsRooms(Room roomTemplet, Enums.EnemyTypes[] procEnemies)
```

Comment:
This class should not be modified.  Its only job is to generate a grid, nothing else.